config/routes.rb

```diff
@@ -0,0 +1,16 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  authenticated :user do
+    root 'categories#index', as: :authenticated_root
+  end
+  unauthenticated do
+    root "splash_screen#index", as: :unauthenticated_root
+  end
+  # root "splash_screen#index"
```

Comment:
- [ Required ] I think it would be better if your code base does not have any pieces of commented code. Commented code creates confusion in your code base and is considered a sign of code that is not clean. Kindly delete this line.